docs/inter.css

```diff
@@ -6,13 +6,15 @@ layout: none
   font-family: 'Inter';
   font-style:  normal;
   font-weight: 100;
+  font-display: 'swap';
```

Comment:
Let's remove the single quotes around `swap` [as per the spec](https://drafts.csswg.org/css-fonts-4/#font-display-desc)

Reply:
Fixed in 3695ee9